Makefile

```diff
@@ -117,8 +117,11 @@ integration: libpodimage
 testunit:
 	$(GO) test -tags "$(BUILDTAGS)" -cover $(PACKAGES)
 
+ginkgo:
+	ginkgo -v test/e2e/
+
 localintegration: test-binaries
-	bash -i ./test/test_runner.sh ${TESTFLAGS}
+	ginkgo -v test/e2e/.
```

Comment:
Don't we need test_runner.sh at least for a little while longer?

Reply:
as i said in the commit message, i disabled the bats tests in ubuntu as we are running them in fedora and centos.

Reply:
I'm OK with it if the intent is the eventual removal of BATS tests as they are replaced by Ginkgo tests.

Reply:
Yes, that is the point ... I removed it in part because the travis tests are so darned long and i wanted to take some burden off the ci.  i can re-enable them but given all the ci issues we have had of late, i think it will will contributory to frustration.

Reply:
added back in per request ...